components/event.jsx

```diff
@@ -14,10 +14,10 @@ const event = ({ name, description, datetime, location }) => (
           {description}
         </Card.Text>
         <FontAwesomeIcon className="mr-1" icon={faClock} fixedWidth />
-        {moment(datetime).format('ddd. MMMM Do, h:mma')}
+         {moment(datetime).format('ddd. MMMM Do, h:mma')}
         <br />
         <FontAwesomeIcon className="ml-1 mr-1" icon={faMapPin} />
-        {location}
+         {location}
```

Comment:
Remove

Reply:
Fixed in f8f64b1992724eb315c202799ff5ecab137011cd